cpp/core/compute/Runtime.h

```diff
@@ -63,26 +63,9 @@ class Runtime : public std::enable_shared_from_this<Runtime> {
   Runtime(const std::unordered_map<std::string, std::string>& confMap) : confMap_(confMap) {}
   virtual ~Runtime() = default;
 
-  void parsePlan(const uint8_t* data, int32_t size) {
-    parsePlan(data, size, {-1, -1, -1});
-  }
-
   /// Parse and cache the plan.
   /// Return true if parsed successfully.
-  void parsePlan(const uint8_t* data, int32_t size, SparkTaskInfo taskInfo) {
-    taskInfo_ = taskInfo;
-#ifdef GLUTEN_PRINT_DEBUG
```

Comment:
Do we want to control debug log through Gluten config in this PR? Previously, it is controlled by the compilation flag, see https://github.com/oap-project/gluten/blob/main/cpp/CMakeLists.txt#L76-L79. Maybe an option is to control the cpp compilation level through scala config.